src/build/config/config.yaml

```diff
@@ -767,8 +767,8 @@ option:
   compress-level-network:
     section: global
     type: integer
-    default: 3
-    allow-range: [0, 9]
+    default: 1
+    allow-range: [-5, 12]
```

Comment:
`[-5, 12]` this meets the boundaries we have in `compress.h`. I've struggled to find this information on the lz4 repo but out of curiosity, why the `-5`?

Reply:
I found these values by digging pretty deep into the lz4 code a few years ago (we are already using these values for compress-level). The max value was fairly easy to find, the min value much less so. Anyway, the negative values enable a special high-speed mode, which could be useful for protocol compression especially.

Reply:
Thanks for the explanation :+1: